src/modules/Workspaces/WorkspacesEditor/Data/ProjectData.cs

```diff
@@ -37,6 +37,8 @@ public struct WindowPositionWrapper
 
             public string AppUserModelId { get; set; }
 
+            public string PwaAppId { get; set; }
```

Comment:
For existing workspaces, when the Editor is opened, the `pwa-app-id` is set to `null`. This causes an "Incorrect workspaces.json file" error in the Launcher.